types/koa-session/index.d.ts

```diff
@@ -194,6 +201,19 @@ declare namespace session {
          */
         destroy(key: string): any;
     }
+
+    interface externalKeys {
```

Comment:
The common styleguide suggests using CamelCase and the other part of this module seems to follow this, see guide:
https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md#interface
```ts
interface ExternalKeys
```